composer.json

```diff
@@ -5,7 +5,7 @@
 	"license": "Apache-2.0",
 	"require": {
 		"pocketmine/pocketmine-mp": "^4.0.0",
-		"sof3/await-generator": "^3.1.1"
+		"sof3/await-generator": "^3.3.0"
```

Comment:
According to [here](https://github.com/SOF3/await-generator/blob/master/CHANGELOG.md#330) v3.3.0 is when await-generator had `Await::promise()` added, which means I had to bump the version in `composer.json`.